src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -130,6 +131,11 @@ export default class GroupComparisonPage extends React.Component<
             this.store.methylationEnrichmentProfiles,
             this.store.survivalClinicalDataExists,
             this.store.genericAssayEnrichmentProfilesGroupedByGenericAssayType,
+            this.store.mutationsByGroup,
+            this.store.availableGenes,
+            this.store.mutations,
+            // this.store.allSamples,
```

Comment:
cleanup

Reply:
await these newly added promises (only used by new tab) inside of new tab so that other tabs don't have to wait for these things